packages/dependency-extraction-webpack-plugin/README.md

```diff
@@ -119,7 +187,7 @@ The filename for the generated asset file. Accepts the same values as the Webpac
 -   Type: boolean
 -   Default: `false`
 
-By default, one asset file is created for each entry point. When this flag is set to `true`, all information about assets is combined into a single `assets.(json|php)` file generated in the output directory.
+By default, one asset file is created for each entry point. When this flag is set to `true`, all information about assets is combined into a single `asset.(json|php)` file generated in the output directory.
```

Comment:
That wasn’t a typo. See 

https://github.com/WordPress/gutenberg/blob/d362529ecc78ae3054e1ee65895b8725020dd0c6/packages/dependency-extraction-webpack-plugin/lib/index.js#L250

Reply:
I think that code has a typo, this outputs `asset.*` by default:

https://github.com/WordPress/gutenberg/blob/d362529ecc78ae3054e1ee65895b8725020dd0c6/packages/dependency-extraction-webpack-plugin/lib/index.js#L230C4-L235

All the core blocks use `*.asset.php` and its what core looks for:

https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks/file/view.asset.php

https://github.com/WordPress/wordpress-develop/blob/e43275b61c6e3a8d5657b6d21eb99bf1bafd52b2/src/wp-includes/blocks.php#L159

Reply:
Oh, I see 🤦 

I'll revert this change.